test/caller-test.jl

```diff
@@ -89,6 +76,19 @@
     @test 0 == reduce(x1*x2*x3*x4*x5*x6*x7*x8*x9 - 1, id)
 end
 
+@testset "caller.LibFinvar" begin
+   Qw, (w,) = FunctionField(QQ, ["w"])
+   K, w = AlgebraicExtensionField(Qw, w^2+w+1)
+   R, (x, y, z) = PolynomialRing(K, ["x","y","z"])
+   M1 = Matrix(R, [0 1 0; 0 0 1; 1 0 0])
+   M2 = Matrix(R, [1 0 0; 0 w 0; 0 0 w^2])
+   P, S, IS = Singular.LibFinvar.invariant_ring(M1, M2)
+   @assert isa(P, Singular.smatrix{Singular.spoly{Singular.n_algExt}})
```

Comment:
Same as above: how about this?
```suggestion
   @assert P ia Singular.smatrix{Singular.spoly{Singular.n_algExt}}
```